etna/insights/migrations/0033_auto_20220304_2226.py

```diff
@@ -0,0 +1,411 @@
+# Generated by Django 3.2.12 on 2022-03-04 22:26
+
+
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    # We get the model from the versioned app registry;
+    # if we directly import it, it'll be the wrong version
+    tag = apps.get_model("taggit", "Tag")
+    db_alias = schema_editor.connection.alias
+    tag.objects.using(db_alias).bulk_create(
+        [
+            tag(name="Africa", slug="Africa"),
+            tag(name="Aid and development", slug="Aid and development"),
```

Comment:
Slugs in Django usually consist of lower-case letters, with words separated by hyphens. There's a util function you can use that will take care of conversion for you: [django.utils.text.slugify](https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.text.slugify)

Using this method you could write something like this:
```
from django.utils.text import slugify

...

to_create = [
    tag(name=v, slug=slugify(v)) for v in (
        "Africa",
        "Aid and development",
        "Air Force",
        ....
    )
]
tag.objects.bulk_create(to_create)
```

Reply:
Yes , very useful, done now!